search.js

```diff
@@ -86,60 +86,44 @@ let pubMedApi = {
 
         pmSvc.fetchSummary(environment, options)
             .then(summaryResults => {
+
+                pmids = summaryResults.result.uids;
+                pmcids = [];
+
+                // digging pmcids out from results
+                for (i = 0; i < pmids.length; i++) { 
+                    for(j = 0; j < summaryResults.result[pmids[i]].articleids.length; j++){
```

Comment:
I think you can probably replace both of these loops with its `forEach` equivalent, right?